src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy

```diff
@@ -304,4 +309,39 @@ class DebugSpec extends BaseSpec {
         null         || 0
         null         || null
     }
+
+    def "PBS should return HTTP call type when it's default bidder call"() {
+        given: "Default basic generic BidRequest"
+        def bidRequest = BidRequest.defaultBidRequest
+
+        when: "PBS processes auction request"
+        def response = defaultPbsService.sendAuctionRequest(bidRequest)
+
+        then: "Response should contain call type HTTP"
+        assert response.ext?.debug?.httpcalls[GENERIC.value].first().calltype == HTTP
```

Comment:
It should be inverted. When it's a regular bidder call, `calltype` field should not be populated at all.